### PR TITLE
feat: Add user verify code sign  type feature

### LINKED ISSUE #7: 登陆注册流程更新
https://www.figma.com/design/WL4amwv0XAHPGrzPT3lDkV/OpenBuild-login-optimization?node-id=0-1&m=dev&t=Gc45VtA35vV8dYEj-1

去掉注册概念，账号验证码以及账号密码登录
image

### PR DESCRIPTION
**Pull request by UstinianMobius**
- New email authentication code sign type
- The SvgIcon component receives the Change event
- Resolves #7

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -116,5 +116,6 @@
     "postcss": "^8.4.18",
     "prettier": "^2.5.1",
     "tailwindcss": "^3.2.4"
-  }
+  },
+  "packageManager": "pnpm@9.15.1+sha512.1acb565e6193efbebda772702950469150cf12bcc764262e7587e71d19dc98a423dff9536e57ea44c49bdf790ff694e83c27be5faa23d67e0c033b583be4bfcf"
```
Comment (ourai) [inline package.json:120]: This field doesn't need to be specified for now.

File: src/app/(uc)/signin/page.js [hunk 2]
```diff
@@ -14,120 +14,251 @@
...
-import Link from 'next/link'
-import { signIn } from 'next-auth/react'
-import { useForm } from 'react-hook-form'
-import { toast } from 'react-toastify'
-import { EyeIcon, EyeSlashIcon } from '@heroicons/react/24/outline'
-
-import { signin } from '#/services/auth'
-import { wrapOnChange } from '@/utils/form'
-import Loader from '@/components/Loader'
+"use client";
```
Comment (ourai) [inline src/app/(uc)/signin/page.js:17]: We use single quote for strings.

File: yarn.lock
Comment (ourai) [inline yarn.lock:1]: The `yarn.lock` file couldn't be deleted, or Vercel build errors will occur.

**Review by ourai — CHANGES_REQUESTED** [review]
The functionality part will be reviewed when the Vercel preview environment is ready.

File: yarn.lock
Comment (UstinianMobius) [reply yarn.lock:1]: The problem has been solved. 
1: The yarn.lock file is retained.If you switch to pnpm later, should change the vercel configuration again. 
2: Format the file with single quotes

**Pull request closed without merging**